src/shared/bg_pmove.cpp

```diff
@@ -51,11 +51,13 @@ struct pml_t
 	vec3_t   previous_origin;
 	vec3_t   previous_velocity;
 	int      previous_waterlevel;
+
```

Comment:
(negligible, but probs a typo)

Reply:
not a typo: it "splits" the struct between properties and methods